cmd/drone-acr/main.go

```diff
@@ -62,3 +263,9 @@ func getenv(key ...string) (s string) {
 	}
 	return
 }
+
+type strct struct {
```

Comment:
Naming? What exactly is this?

Reply:
This code was taken from `drone-kaniko`: https://github.com/drone/drone-kaniko/blob/main/cmd/kaniko-acr/main.go#L442

Once the implementation is approved, I can make this change and take approval from US team and merge this PR.